README.md

```diff
@@ -1 +1,60 @@
-# my-portfolio
\ No newline at end of file
+# my-portfolio
+![](https://img.shields.io/badge/myapp-blueviolet)
+
+
+> Personal portfolio project.
+
+<!-- ![screenshot](./app_screenshot.png) -->
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## Live Demo
+
+[Live Demo Link](https://github.com/dami1080/my-portfolio/tree/features)
```

Comment:
- [ ] Please fix the Live Demo Link or remove its section from the README since it's not a requirement for this project.
Remember that broken links are very small mistakes but they might leave a bad impression on future viewers.